src/pages/api/register.ts

```diff
@@ -0,0 +1,95 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import Auth from '@aws-amplify/auth';
+import { getDomain, redirectTo, redirectToError, setCookieOnResponseObject } from './apiUtils';
+import { USER_COOKIE } from '../../constants';
+import { InputCheck } from '../register';
+import { getServicesByNocCode } from '../../data/auroradb';
+
+const checkEmailValid = (email: string) => {
+    const emailRegex = new RegExp('^[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,4}$');
+    return emailRegex.test(email) && email !== '';
```

Comment:
I think this regex wont allow whitespace at the front, meaning ' test@email.com' will be invalid.

Reply:
cognito seems to be handling this

Reply:
have tested this and seems cognito deals with this